src/network/taraxa_capability.cpp

```diff
@@ -576,7 +576,7 @@ void TaraxaCapability::interpretCapabilityPacketImpl(NodeID const &_nodeID, unsi
         std::vector<Vote> send_next_votes_bundle;
         for (auto const &v : next_votes_bundle) {
           if (!peer->isVoteKnown(v.getHash())) {
-            send_next_votes_bundle.emplace_back(v);
+            send_next_votes_bundle.push_back(std::move(v));
```

Comment:
I think emplace_back(v) is some kind same with push_back(std::move(v)). What's different with them?

Reply:
so emplace should be used when you want to create object in place and insert it to the container. Here we want to move that object so we need to do `push_back(std::move(v))` or `emplace_back(std::move(v))` this should be same, but I prefer to use emplace only when we creating specific object

Reply:
Thanks!